src/dto/EmployeeDto.java

```diff
@@ -1,4 +1,54 @@
 package dto;
 
+import java.util.Date;
+
 public class EmployeeDto {
+    private String name;
+    private String surname;
+    private String occupation;
+    private Date date;
+    public DepartmentDto getDepartment() {
+        return department;
+    }
+
+    public void setDepartment(DepartmentDto department) {
+        this.department = department;
+    }
+
+    private DepartmentDto department;
+
+    private int numberOfExperience;
```

Comment:
what is the numberOfExperience ? Why do you need it ? I did not saw any requirement for this